src/test/java/com/microsoft/azure/spring/data/cosmosdb/repository/integration/RoleRepositoryCollectionIT.java

```diff
@@ -37,6 +45,11 @@
     @Autowired
     private CosmosTemplate template;
 
+    @Before
+    public void setup() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://app.codacy.com/manual/Microsoft/spring-data-cosmosdb/pullRequest?prid=4925697)